src/main/java/uk/gov/hmcts/reform/wataskmanagementapi/domain/entities/task/Task.java

```diff
@@ -113,10 +113,14 @@
     )
     private boolean autoAssigned;
 
-    @ApiModelProperty(required = false,
+    @ApiModelProperty(required = true,
         notes = "boolean to show if a warning is applied to task by a service task in a subprocess")
     private Boolean hasWarnings;
 
+    @ApiModelProperty(required = true,
```

Comment:
could this be a breaking change for any other team consuming our API?

Reply:
fixed